### PR TITLE
[PLAT-803] jfrogify

### PR DESCRIPTION
**Pull request by deathweaselx86**
This updates the package name in setup.py to not conflict with the main django-health-check pypi package. It also updates the README.md to reflect how to add the requirement.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,18 +28,11 @@ Supported Versions
 
 Installation
 ------------
-
-First, add the requirement to your `requirements-without-pypi.in` file, using the latest tagged release:
+First, add the requirement to your `requirements.in` file, using the latest tagged release:
 ```
 -e git://github.com/insurancezebra/django-health-check.git@4.1.2#egg=django-health-check
```
Comment (tony-zebra) [inline README.md]: This line needs to change, right?

Comment (deathweaselx86) [reply README.md]: I don't know why that change didn't go in

Comment (deathweaselx86) [reply README.md]: @tony-zebra fixed

Comment (tony-zebra) [reply README.md]: Sorry, should have been more specific.  I mean shouldn't line 33 be `thezebra-django-health-check=='4.1.2'`

Comment (deathweaselx86) [reply README.md]: It is now?

**Pull request merged**